package.json

```diff
@@ -15,16 +15,19 @@
   ],
   "scripts": {
     "setup": "wsrun -tm setup",
-    "build": "wsrun -tm -x vod-to-live-service build",
-    "clean": "wsrun -tm -x vod-to-live-service clean",
+    "build": "wsrun -tm -x vod-to-live-service -x channel-service build",
```

Comment:
`channel-workflows` are currently included into `build` and `clean` scripts,
should be excluded for consistency